src/do_wear.c

```diff
@@ -549,6 +549,7 @@ boolean voluntary; /* taking gloves off on purpose? */
         instapetrify(kbuf);
         /* life-saved; can't continue wielding cockatrice corpse though */
         remove_worn_item(obj, FALSE);
+        tnnt_achieve(A_CURED_STONING);
```

Comment:
This one is debatable (is it really "surviving" stoning if you die and then come back, and have lost the survivor conduct?), and in any case this is the wrong way to go about it since there are so many points in the code where you can be petrified, of which this is only one.

Reply:
i'll drop this out, then. i was just going off the bugreport listed on etherpad.